pkg/stats/collector.go

```diff
@@ -217,8 +223,8 @@ func (s *BufferedCollector) getInstallationID() string {
 	return s.installationID
 }
 
-func (s *BufferedCollector) incr(k Event) {
-	s.cache[k]++
+func (s *BufferedCollector) incr(k EventWithCount) {
```

Comment:
As I see it we cloud update `Event` to include count. But we added `EventWithCount` - Any reason?
In this case I'll keep `incr` current signature that accepts `Event` and increment just 1 and add `add` or `update` method that will accept `EventWithCount`.

Reply:
done